src/main/java/com/mewna/catnip/Catnip.java

```diff
@@ -116,6 +123,67 @@ static String getShardCountUrl() {
     @Nullable
     User selfUser();
     
+    default void setPresence(@Nonnull final Presence presence) {
+        setPresence(presence, true);
+    }
+    
+    default void setPresence(@Nonnull final Presence presence, final boolean broadcastToAllShards) {
+        if (!broadcastToAllShards) {
```

Comment:
Sending to a random shard feels like a bad idea since end-users probably wouldn't expect it. I would think that just having "set presence for specific shard" and "set presence for all shards" would be enough, and "set for all shards" could just be a loop over "set for specific shard"